packages/@lwc/style-compiler/src/postcss-lwc-plugin.ts

```diff
@@ -30,7 +30,10 @@ function selectorProcessorFactory(transformConfig: SelectorScopingConfig) {
     });
 }
 
-export default function postCssLwcPlugin(options: { scoped: boolean }): TransformCallback {
+export default function postCssLwcPlugin(options: {
+    scoped: boolean;
+    apiVersion: APIVersion;
```

Comment:
Not seeing any usages of this variable in this function. Is it being added for a future usecase?

Reply:
Yes exactly; it's not used for anything right now. I just want to make it easy for folks in the future to add API versioning to the style compiler.